### PR TITLE
fix: plugin for dist and popover modifiers

### PR DESCRIPTION
**Pull request by Harris-YWang**
Hot fix for `data-testid` is still in dist files

Fix modifiers not able to be overwritten

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/components/Popover/Popper.jsx [hunk 1]
```diff
@@ -64,22 +62,25 @@ const Popper = ({
...
 
-  const { styles, attributes, update } = usePopper(refElement, popperElement, {
-    modifiers: [
-      { name: 'arrow', options: { element: arrowElement } },
-      {
-        name: 'offset',
-        options: {
-          offset: [0, 6],
-        },
+  // react-popper not using subsequent settings to overwrite preceding ones
```
Comment (Harris-YWang) [inline src/components/Popover/Popper.jsx:65]: ^ If the modifiers is `[{name:'flip', options:{}}, {name:'flip', options:{altBoundary: true}}]`, the second object is not working. So, set up default modifiers only if there is no explicitly defined by users

**Review by superpower1 — APPROVED** [review]

**Review by vinteo — APPROVED** [review]

**Comment by bchew** [comment]
Merging :rocket:

File: src/components/Popover/Popper.jsx [hunk 2]
```diff
@@ -64,22 +62,25 @@ const Popper = ({
...
+    });
+  if (!_.find(modifiers, { name: 'flip' }))
+    defaultModifiers.push({
+      name: 'flip',
+      options: {
+        altBoundary: true,
       },
-    ].concat(modifiers),
+    });
+
```
Comment (superpower1) [inline src/components/Popover/Popper.jsx:81]: You can change this part to:
```
_(modifiers).keyBy('name').merge(defaultModifiers).values().value()
```
which will allow user to partially change the default modifier

Comment (Harris-YWang) [reply src/components/Popover/Popper.jsx:81]: not necessary but could be an improvement, please create a new pr if still want it

**Pull request merged**